docs/polaris/queueing-and-running-jobs/example-job-scripts.md

```diff
@@ -76,26 +110,33 @@ cd ${PBS_O_WORKDIR}
 # MPI and OpenMP settings
 NNODES=`wc -l < $PBS_NODEFILE`
 NRANKS_PER_NODE=$(nvidia-smi -L | wc -l)
-NDEPTH=8
+NDEPTH=16
```

Comment:
I don't understand this change. It seems that most of the physical cores are unused but it tries to use hardware threads potentially sharing the same physical cores.

Reply:
I changed `--cpu-bind` to `numa`, so for example the second `mpiexec` would return:
```
“RANK= 1 LOCAL_RANK= 1 gpu= 2”
“RANK= 3 LOCAL_RANK= 3 gpu= 0”
“RANK= 2 LOCAL_RANK= 2 gpu= 1”
“RANK= 0 LOCAL_RANK= 0 gpu= 3”
rnk= 0 :  # of devices detected= 1
To affinity and beyond!! nname= x3110c0s37b1n0  rnk= 0  list_cores= (0-7,32-39)  num_devices= 1
    [0,0] Platform[ Nvidia ] Type[ GPU ] Device[ NVIDIA A100-SXM4-40GB ]  uuid= GPU-13d5a2c5-8133-f886-318f-5d2d8a423f98
To affinity and beyond!! nname= x3110c0s37b1n0  rnk= 1  list_cores= (8-15,40-47)  num_devices= 1
    [1,0] Platform[ Nvidia ] Type[ GPU ] Device[ NVIDIA A100-SXM4-40GB ]  uuid= GPU-4ee60148-1c9a-0fcb-f3d6-5e0a95fb2f45
To affinity and beyond!! nname= x3110c0s37b1n0  rnk= 2  list_cores= (16-23,48-55)  num_devices= 1
    [2,0] Platform[ Nvidia ] Type[ GPU ] Device[ NVIDIA A100-SXM4-40GB ]  uuid= GPU-a2b7c47f-0ab8-2a38-afa7-be04d3284884
To affinity and beyond!! nname= x3110c0s37b1n0  rnk= 3  list_cores= (24-31,56-63)  num_devices= 1
    [3,0] Platform[ Nvidia ] Type[ GPU ] Device[ NVIDIA A100-SXM4-40GB ]  uuid= GPU-1eaaeaa5-342d-7da5-a674-d6300b54f8fa
```